general/1_terminal/Danylo_Kulbachinskiy/script1.sh

```diff
@@ -0,0 +1,24 @@
+touch zoola1.txt
+touch zoola2.txt
+touch zoola3.txt
+touch zoom.txt
+touch hello.txt
+touch hizoola.txt
+
+set -e
```

Comment:
Good job on reading about `set -e`, this is a good thing to use in your scripts.